packages/graphql-server/src/directives/authDirectives.ts

```diff
@@ -0,0 +1,25 @@
+import gql from 'graphql-tag'
+
+import { AuthenticationError } from '../errors'
+
+export const REQUIRE_AUTH_SDL = /* GraphQL */ `
+  directive @requireAuth(roles: [String]) on FIELD_DEFINITION
+`
+
+// onExecute
+export const requireAuth = () => {
```

Comment:
This is where we would bring in the isAuthenticated() and hasRoles() from the app's auth.js ... maybe in api/auth?

Needs to support roles, too.